pyrene/src/components/TabView/TabView.examples.jsx

```diff
@@ -1,18 +1,27 @@
 import React from 'react';
+import Icon from '../Icon/Icon';
 import Placeholder from '../../examples/Placeholder';
 
 const TabView = {};
 
+const nameRenderCallback = (name) => (
+  <div style={{ marginLeft: 4 }}>
```

Comment:
If you d like to, you can horizontally align the tab label and the icon along.

`<div style={{ marginLeft: 4, position: 'relative', top: -2 }}>`


![Pyrene](https://user-images.githubusercontent.com/6510794/116210498-380dad80-a743-11eb-9180-197268d675b8.png)